examples/balrog/babyai/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+# This is a simplified example for learning purposes. Do not use this in production.
+# For production-ready deployments, see: https://www.tensorzero.com/docs/gateway/deployment
+
+services:
+  clickhouse:
+    image: clickhouse/clickhouse-server
+    ports:
+      - "8123:8123"
+    healthcheck:
+      test: wget --spider --tries 1 http://localhost:8123/ping
+      start_period: 30s
+      start_interval: 1s
+      timeout: 1s
+
+  gateway:
+    image: tensorzero/gateway
+    volumes:
+      # Mount our tensorzero.toml file into the container
+      - ./config:/app/config:ro
+    environment:
+      - CLICKHOUSE_URL=http://tensorzero-clickhouse-babyai:8123/tensorzero
```

Comment:
this doesn't work as-is. I think you need to add `container_name: tensorzero-clickhouse-babyai` to the docker compose here.

Reply:
then for consistency you will want to add `container_name: tensorzero-gateway-babyai` to the gateway service